src/providers/config/config.ts

```diff
@@ -222,7 +222,7 @@ export class ConfigProvider {
         buycrypto: true,
         exchangecrypto: true,
         giftcards: true,
-        walletConnect: true
+        walletConnect: false
```

Comment:
Is it okay for walletConnect to be turned off by default?

Reply:
Yes. We don't know that it shows in Home by default.